reproman/interface/ls.py

```diff
@@ -100,4 +109,10 @@ def __call__(resrefs=None, verbose=False, refresh=False):
             manager.save_inventory()
         else:
             ui.message('Use --refresh option to view updated status.')
+
+        if unknown_resrefs:
+            raise ResourceNotFoundError(
+                "Could not find the following resources: {}"
+                .format(unknown_resrefs))
```

Comment:
```suggestion
                .format(' '.join(unknown_resrefs)))
```

Reply:
I'd prefer to just format the list as is, but given you feel strongly enough to suggest a `.join`, I'll make that change.  I think space is a bad delimiter though.  I'll go with ', '.